### DISCUSSION TITLE
Kibana visualization errors in SO 2.3.40

### DISCUSSION THREAD

**Discussion by cyb3rslack3r** [Security-Onion-Solutions/securityonion#3979] [Unsupported Versions] [answered]
I installed an older version of SO 2.. and found grid displayed the data differently if I ran soup before adding all the sensors to my distributed deployment.  The sensors added after the upgrade to SO 2.3.40 did not show an IP address making me think they weren't working correctly.  Everything went fine during installation if it occurred pre soup (all managed by the SO MGR), but sensors installed after running SOUP on MGR would get an error towards the end of install.  Don't remember the exact error and don't have the data in front of me, but when I queried the sosetup.log I believe it mentioned errors with SQL.  

No matter how I got to version 2.3.40 (upgrade or directly installed the 2.3.40 iso) in my distributed deployment, there was always several errors in Kibana. The distributed deployment is 2 Sensors, 3 Search nodes  and 1 manager. The Kibana errors were: "Could not locate that index-pattern-field (id: pick.a.dashboard)" in severals dashboards, but not all.  When I setup the manager I did not install OSQuery or Wazuh, because they are strictly for network traffic.  I have tried all the fixes seen here with no success and the only fix was installing 2.3.1 and not upgrading.  Is the only answer to wait for the next version after 2.3.40 to see if this eliminates the problem in a distributed deployment? Thanks

**Comment by weslambert** [comment, marked as answer]
I would wait until the next release, which should be out very soon -- otherwise, you could try a fresh install from 2.3.40.

**Reply by cyb3rslack3r** [reply]
I tried a fresh install of 2.3.40 and still had problems.  Saw the new version just released so I will try that and open a new issue if this still occurs.  Thanks
